HTML Forms (Sign up page of mint.com)

bini-i opened the pull request:
HTML form sign up page that matches the appearance of mint.com’s signup page

Added the following components
- header with a sign in button
- intuit banner
- form with appropriate input elements
- footer with copyright description
- media query where the header collapses for width less than 700px

Note: the whole page is responsive

------------modifications after review-----------------
- removed inline attributes
- replaced "a" tag with appropriate input tag

bini-i committed:
intialized project folder with html and css files and add semantic structures

wintan1418 committed:
i worked on the header,and i also used the css reset and styled the three elements i worked on the header

bini-i committed:
added a style for link tags
added a form with input elements and labels and styled them

wintan1418 committed:
i added the intuit lofo,and the icons,and also i added the footer and inputed the contents as shown in the clone page given to us

bini-i committed:
Add style to header elements
Improvement

give Roboto font and adjusted the font size

bini-i committed:
Update styling of form
Improvement
Update colors, alignment and font-size

bini-i committed:
Update styling to the footer
Improvment
update color of fonts and alignment of elements

wintan1418 committed:
add styling for the sign-up banner,add stylings and allignment to the icons

bini-i committed:
Update banner style and Add linters
update layout of banner
add linters configuration

bini-i committed:
Update README file

- Used template to provide description of the project and authors

bini-i committed:
Add media query

for width less than 700px the header collapses

bini-i committed:
update README.md

bini-i committed:
Fix html to meet linters' requirement

-changed button types

bini-i committed:
Merge branch 'sign_up_page' of github.com:bini-i/HTML-Forms into sign_up_page

bini-i committed:
Fix index.html for linters' requirement

-add button types

bini-i committed:
Fix html page for linters' requirement

- removed crossorigin from script
- changed button type to submit

bini-i committed:
Fix style for linters' requirement

- adjusted spacings

bini-i committed:
Fix html page for linters' requirement

-removed a tag from being descendent of p tag

bini-i committed:
Fix html page based on linters' requriment

javitocor commented on index.html:
```diff
@@ -0,0 +1,86 @@
+    </header>
+    <main>
+      <div class="intuit_logo">
+          <img src="images/intuit_logo.png" alt="intuit logo" width="130">
```
Do not include inline styling, kindly remove it!
Make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

javitocor commented on index.html:
```diff
@@ -0,0 +1,86 @@
+
+      <div class="icons">
+        <div class="icon">
+          <img src="images/icon1.png" alt="mint-logo" width="25" />
```
Same here!

javitocor commented on index.html:
```diff
@@ -0,0 +1,86 @@
+          <p>mint</p>
+        </div>
+        <div class="icon">
+          <img src="images/icon2.svg" alt="mint-logo" width="25" />
```
Same here!

javitocor commented on index.html:
```diff
@@ -0,0 +1,86 @@
+          <p>quickbooks</p>
+        </div>
+        <div class="icon">
+          <img src="images/icon3.png" alt="mint-logo" width="25"/>
```
Same here!

javitocor commented on index.html:
```diff
@@ -0,0 +1,86 @@
+        <label for="conf_password">Confirm Password</label>
+        <input type="password" id="conf_password" name="email" value="">
+
+        <a class="submit_button" href="#"><i class="fa fa-lock" aria-hidden="true"> </i>Create Account</a>
```
Your form should have button or an input tag type submit, modify the actual a tag

It has a `<button>` or an `<input>` of type `submit` that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

bini-i replied:
Thank you for your feedback,
do I have to make a new pull request, after I made modifications?

javitocor reviewed (changes requested):
# CHANGES REQUIRED ♻️ 

Amazing job in this project!!! I am impressed! 👍 
The website you made is a real clone! you could do some fishing with it! 😆  But don't do! 🛑 

However, there are small details that must be modified in order to approve the project ▶️ , nothing that you cannot achieve 💪 

Check my comments below 👇 

Once you have made the required changes, resubmit the project for a new code review ⏩ 

Good luck with it! 😃 

If you have any question or doubt you can reach me out in Slack

bini-i committed:
Fix index.html

- removed inline styling
- modified submit button into 'input' tag from 'a' tag




issue).

javitocor reviewed (approved):
# PROJECT APPROVED 🥇 

Congratulations team!! 👏 

You have met all the requirements, the project is approved ⭐ 

Remember to fill up the completion form in your dashboard 📑  and move to the next project ⏩ 

Good luck, guys 😃
